PackageExplorer/PackageViewer.xaml.cs

```diff
@@ -626,6 +625,7 @@ private bool HandleDataObject(PackageFolder? folder, IDataObject data, bool copy
             return false;
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "<Pending>")]
```

Comment:
What's "pending" means here?

Reply:
that's the default comment put in by the suppression gesture.